browser/src/layer/vector/SVGGroup.js

```diff
@@ -174,7 +174,7 @@ L.SVGGroup = L.Layer.extend({
 	_onDragStart: function(evt) {
 		if (!this._map || !this._dragShapePresent || !this.dragging)
 			return;
-		if (this._map._docLayer._cursorMarker && this._map._docLayer._cursorMarker.isVisible())
+		if (this._map._docLayer._cursorMarker && this._map._docLayer._cursorMarker.isVisible() && this.options.isText)
```

Comment:
what does this new bit do?

Reply:
if we return from here, it prevents drag animation and allows us to show selection. this new isText option also new and is added in this PR.

Reply:
#7821 (which is I believe the same issue) also applies to non text objects. When I had a look at it it's where I found the code path did diverge.